dev-python/ciso8601/metadata.xml

```diff
@@ -17,9 +20,6 @@
     popular subset. Supported subsets: 
     https://github.com/closeio/ciso8601#supported-subset-of-iso-8601
   </longdescription>
-  <use>
-    <flag name="test">Enable test support</flag>
```

Comment:
Why not use the global "test" USE flag?

Reply:
of course, good idea :) thanks!

Reply:
repoman or pkgcheck should have told you :-)